conda-store-server/conda_store_server/_internal/server/templates/conda-store-ui.html

```diff
@@ -28,8 +28,8 @@
              REACT_APP_URL_BASENAME: new URL("{{ url_for('get_conda_store_ui') }}").pathname
          };
         </script>
-        <script defer src="static/conda-store-ui/main.js"></script>
-        <link href="static/conda-store-ui/main.css" rel="stylesheet">
+        <script defer src="{{ url_for('get_conda_store_ui') }}static/conda-store-ui/main.js"></script>
+        <link href="{{ url_for('get_conda_store_ui') }}static/conda-store-ui/main.css" rel="stylesheet">
```

Comment:
Adding `url_for('get_conda_store_ui')` was the only way I could figure out how to put the url_prefix here. 

Why is this change needed? Previously, the server would only serve the UI app at `url_prefix` (e.g., either at "/" or "/conda-store", depending on configuration).

However, this PR allows the UI app to be loaded at /conda-store/default/test, for example. At that URL, this page would try to fetch the JavaScript and CSS at:

- /conda-store/default/static/conda-store-ui/main.js
- /conda-store/default/static/main.css

These routes end up matching the catch-all, which means the server serves this template page for those requests instead of the desired JavaScript and CSS.